tm4c123x-hal/src/lib.rs

```diff
@@ -25,8 +25,10 @@
 
 pub use tm4c_hal::bb;
 pub use tm4c_hal::time;
+pub use tm4c_hal::delay;
+
+pub use tm4c_hal::gpio as gpio_common;
```

Comment:
Is this basically a duplicate of the `pub use tm4c_hal::gpio::*` in gpio.rs?

Reply:
Yes, I think it was from futzing around trying to get the gpio_macro! to import from tm4c-hal::gpio. I gave up in the end and moved the macro to the tm4c-hal lib.rs.

Now removed.